Hive.NET.Core/Configuration/HiveBootstrapper.cs

```diff
@@ -1,12 +1,20 @@
 ﻿using Hive.NET.Core.Manager;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace Hive.NET.Core.Configuration;
 
 public static class HiveBootstrapper
 {
-    public static void ConfigureHive(this IServiceCollection services)
+    private const string HiveSectionName = "Hive";
+    public static void ConfigureHive(this IServiceCollection services, IConfiguration configuration)
     {
         services.AddSingleton<IHiveManager, HiveManager>();
+        services.Configure<HiveSettings>(configuration.GetSection(HiveSectionName));
+    }
+
+    public static void UseHive(this IServiceProvider serviceProvider)
```

Comment:
maybie it can return IServiceProvider for using it as a cascade like .Use().Use()

Reply:
Oh yeah, good point :)

Reply:
Done